onnxruntime/test/optimizer/graph_transform_test.cc

```diff
@@ -7642,5 +7643,87 @@ TEST_F(GraphTransformationTests, GatherToSliceFusion) {
   }
 }
 
+TEST_F(GraphTransformationTests, GatherSliceToSplitFusion) {
```

Comment:
add a test to cover different dims?

also, add a test to cover the case when your optimizer can't optimize it, for example gather in diff dims

Reply:
Updated, thanks!!, please check the unitests.